### DISCUSSION TITLE
Do I need endlessh-go?

### DISCUSSION THREAD

**Discussion by PrzemekSkw** [shizunge/endlessh-go#140] [General]
Hello, I use ssh to my unraid server only in local network or with tailscale vpn. I have few apps exposed to internet with nginx proxy manager. Can somebody tell me if I need endlessh to secure my server?
Regards.

**Comment by sym0nd0** [comment]
Even though you asked this nearly a year ago, I'll still answer.

No one _needs_ endlessh-go (or the OG C version), it's just a fun way to mess with people that are scanning for open SSH ports and wasting their time and resources.

If you have SSH accessible from the internet, which it sounds like you don't, then you need to have hardened your server. A part of that for me is changing the port I access it through and so I have endlessh running on the default port (22) for a laugh.

**Comment by PrzemekSkw** [comment]
Thanks for answer.
Regards.
